CODEOWNERS

```diff
@@ -6,9 +6,13 @@
 
 packages/android_alarm_manager/* @bkonyi
 packages/android_intent/* @mklim
+packages/android_intent/* @matthew-carroll
 packages/battery/* @amirh
+packages/battery/* @matthew-carroll
 packages/camera/* @bparrishMines @mklim
 packages/connectivity/* @cyanglaz
+packages/connectivity/* @matthew-carroll
```

Comment:
It seems more common (and consistent with this file) to put all the CODEOWNERs on one line. See this example:

https://github.com/python/cpython/blob/master/.github/CODEOWNERS

Reply:
Ok, I placed names on single lines and I also added consistent spacing to more easily read the names like the file you linked.